docs/source/torch.rst

```diff
@@ -258,6 +258,8 @@ Other Operations
 .. autofunction:: einsum
 .. autofunction:: flip
 .. autofunction:: histc
+.. autofunction:: bincount
+.. autofunction:: meshgrid
```

Comment:
These are not in alphabetical order, could you please add them in the right order? The rendered docs don't sort them, unfortunately. Thanks.

Reply:
done :)

Reply:
Thank you. :smile: